app/lib/midnightSpanCorrector.js

```diff
@@ -32,9 +32,19 @@ function setNextCloseToTomorrow(openingTimes, status) {
   return status;
 }
 
+function correctClosedForOpeningSpansMidnight(status) {
+  if (status.isOpen === false && status.moment.format('HH:mm') === '23:59') {
+    /* eslint-disable no-param-reassign */
+    status.isOpen = true;
+    status.nextOpen = status.moment;
```

Comment:
As we're setting the service to open, the next open time will be wrong.
The value is not currently used for an open pharmacy when setting the message, but setting it to the time of the request will prevent problems if used in the future.

Reply:
This is used by the web app https://github.com/nhsuk/connecting-to-services/blob/master/app/lib/addBankHolidayMessage.js#L9 to determine if the next time the place opens is a bank holiday. This change will break that.

Reply:
On second thoughts, this change will be ok as the web app checks whether the place is open - which it is.